src/components/TaskCard/index.jsx

```diff
@@ -1,30 +1,66 @@
-import React from "react";
+import React, { useEffect, useState } from "react";
 import PropTypes from "prop-types";
+import classNames from "classnames";
 import { useDispatch } from "react-redux";
-import { deleteTask } from "store/actions";
-import { ICON_DELETE, DELETE_ICON_ALT_TEXT } from "utils/constant/images";
+import { deleteTask, completeTask } from "store/actions";
+import {
+  ICON_COMPLETE,
+  COMPLETE_ICON_ALT_TEXT,
+  ICON_DELETE,
+  DELETE_ICON_ALT_TEXT,
+} from "utils/constant/images";
+import { getDateDifference } from "utils/helpers/getDateDifference";
 import "components/TaskCard/index.scss";
 import { checkDateString } from "utils/helpers/propCustomValidation";
 import { dateFormatter } from "utils/helpers/dateFormatter";
 
-const TaskCard = ({ id, task, createdTime }) => {
+const TaskCard = ({ id, task, createdTime, completed }) => {
+  const [taskCompletedIn, setTaskCompletedIn] = useState(null);
+
+  const TaskText = classNames({
```

Comment:
CamelCased names are conventionally used for component names. Please use casing accordingly to avoid confusion. 

```suggestion
  const taskTextStyle = classNames({
```